lang/python/pytest/Makefile

```diff
@@ -0,0 +1,49 @@
+#
+# Copyright (C) 2019 CZ.NIC, z. s. p. o. (https://www.nic.cz/)
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=pytest
+PKG_VERSION:=4.4.0
+PKG_RELEASE:=1
+
+PKG_SOURCE:=pytest-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://files.pythonhosted.org/packages/source/p/pytest/
+PKG_HASH:=f21d2f1fb8200830dcbb5d8ec466a9c9120e20d8b53c7585d180125cce1d297a
+
+PKG_MAINTAINER:=Jan Pavlinec <jan.pavlinec@nic.cz>
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE
+
+include $(INCLUDE_DIR)/package.mk
+include ../python3-package.mk
+
+define Package/python3-pytest
+  SUBMENU:=Python
+  SECTION:=lang
+  CATEGORY:=Languages
+  TITLE:=pytest
+  URL:=https://docs.pytest.org/en/latest/
+  DEPENDS:= \
+	+python3-light \
+	+python3-setuptools \
+	+python3-atomicwrites \
+	+python3-more-itertools \
+	+python3-py \
+	+python3-six \
+	+python3-pluggy
```

Comment:
Looking at [`setup.py`](https://github.com/pytest-dev/pytest/blob/4.4.0/setup.py#L5-L17), it looks like pytest also needs attrs. Are you sure this package works without it?

Reply:
I decided to remove xdist from this PR